pkg/apis/aci/v1/snatpolicy_types.go

```diff
@@ -7,8 +7,9 @@ import (
 type PolicyState string
 
 const (
-	Ready  PolicyState = "Ready"
-	Failed PolicyState = "Failed"
+	Ready            PolicyState = "Ready"
+	Failed           PolicyState = "Failed"
+	IpPortsExhausted PolicyState = "IpPortsExhausted"
```

Comment:
We should have ideally kept the status as "Failed" state and added another field to indicate what failed. We can revisit this.